tasks/ebpf.py

```diff
@@ -7,11 +7,7 @@
 import sys
 
 from invoke import task
-
-try:
-    from tabulate import tabulate
-except ImportError:
-    tabulate = None
+from tabulate import tabulate
```

Comment:
The reason for doing the old approach was so that we do not add requirements for packages not used in the CI, to keep the size of the docker image small.

Reply:
Just saw you added this back.